bin/solidus_compare

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+PATHS = [
+  'app/controllers',
+  'app/helpers',
+]
+
+REMOTE_REPO = 'https://github.com/solidusio/solidus.git'
+REMOTE_SOURCE = 'solidus'
+
+if `git remote`.split("\n").none? REMOTE_SOURCE
+  `git remote add -f #{REMOTE_SOURCE} #{REMOTE_REPO}`
+  puts('error with git remote add') & exit if $?.exitstatus != 0
+
+  `git remote update`
+  puts('error with git remote update') & exit if $?.exitstatus != 0
+end
+
+diff = PATHS.map do |path|
+  list = `git diff master:#{path} remotes/#{REMOTE_SOURCE}/master:frontend/#{path} --name-only`
+  puts("error with git diff #{path}") & exit if $?.exitstatus != 0
+
+  next if list.empty?
+
+  puts "\n> #{path} changed files"
+  puts list
+  path
+end.compact
+
+if diff.any?
+  path = diff.first
+  cmd = "git diff master:#{path} remotes/#{REMOTE_SOURCE}/master:frontend/#{path}"
+  puts "\n#{cmd}\n  to check the changes"
+end
```

Comment:
What about pushing the output to a temp file, that we can also use to push the output of the diff in the [CircleCI artifacts tab](https://circleci.com/docs/2.0/artifacts/#artifacts-overview)?

Reply:
Nice idea 👍 
I'm still looking for an easy to read format for differences. But we can start with just a diff a see how it goes